MainWindow/Main.cs

```diff
@@ -2470,7 +2470,7 @@ private void clickExportSAV(object sender, EventArgs e)
             if (SAV.HasBox)
                 SAV.CurrentBox = CB_BoxSelect.SelectedIndex;
 
-            bool dsv = Path.GetExtension(main.FileName).ToLower() == ".dsv";
+            bool dsv = Path.GetExtension(main.FileName)?.ToLower() == ".dsv";
```

Comment:
I don't think Path.GetExtension will return null.  If anything, it would return an empty string, which can still have ToLower called